.eslintrc

```diff
@@ -1,122 +1,4 @@
 {
   "parser": "babel-eslint",
```

Comment:
should it be included everywhere?

Reply:
you did it :) if'ts present in stripes-acq - no